cli/command/signup/cmd.go

```diff
@@ -0,0 +1,68 @@
+package signup
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/cli/command"
+	"github.com/spf13/cobra"
+)
+
+var (
+	runUse     = "signup"
+	runShort   = "Signup for Sauce Labs"
+	runLong    = `TODO: Some long description about signup`
```

Comment:
`run` command had a sort of filler text too. I'm not sure where this would be visible or if we care yet?